PublicMapping/districtbuilder-classic
# Update deployment docs

## Overview

Add a sample `.tfvars` file with example values, and add a reference to `DB_STATE` to the deployment docs.

### Checklist

~- [ ] Files changed in the PR have been `yapf`-ed for style violations~

## Testing Instructions

 * verify that the `sample.tfvars.example` file has all of the values in `variables.tf`, and ensure README changes are correct.

Closes #551
